### DISCUSSION TITLE
Battery Install Capacity

### DISCUSSION THREAD

**Discussion by BombusAlpinus** [wills106/homeassistant-solax-modbus#596] [Q&A] [answered]
Hi, how does the integration calculate the Battery Install Capacity? 
I'm running a rig with a netto capacity of 12,8 kWh and the integration shows "only" 9 kWh. 
Does this affect the calculation of the Battery Fill Status?
My inverters model: X3-Hybrid-10.0-E

![image](https://github.com/wills106/homeassistant-solax-modbus/assets/28042236/ecb3ea68-83bc-4363-9dc1-f4d6c7cf12a3)

Thanks!
Regards,
BA

**Comment by wills106** [comment]
What batteries do you have installed the Tripple Power T63?

With my T45 it scales, so when I had 2 batteries it report 9kWh and then I went to 3 it's 13.5kWh
I'm just wondering if it's internally fixed to calculate based on the T45?

It's a fixed value, it doesn't increase or decrease based on the State of Charge. So doesn't really provide any value, that's why it's disabled by default.

**Comment by BombusAlpinus** [comment]
ok. i see. these are just "cosmetic" values based on a specific type of battery. in your case the solax ones right? 
i'm using two batteries from a different manufacturer with 14,2 kWh brutto and 12,8 kWh netto capacity.  
maybe a future feature? defining number of batteries and capacity for some calculations? or is this already possible?

**Reply by wills106** [reply, marked as answer]
> these are just "cosmetic" values based on a specific type of battery. in your case the solax ones right?

It looks like that's how it works. They were dropped on the Gen4. I was expecting them to display the remaining capacity but they are static.

What sort of calculations do you want to do?

The BMS side of SolaX doesn't really provide much info other than the Charge level and temps.

There is a bit of a discussion on Battery Cycle here https://github.com/wills106/homeassistant-solax-modbus/discussions/87 if that's something you are after.

**Reply by BombusAlpinus** [reply]
ok, thanks for the info... will read the thread you mentioned ... and thanks for the work on this solax modbus integration. 
i'm really appreciating it!
